### PR TITLE
updated changelog 0.27 (larray-editor)

### PR DESCRIPTION
**Pull request by alixdamman**

### CODE REVIEW

**Comment by gdementen** [comment]
Something is awfully wrong with the tests. I wonder if it's not Pandas which did a subtle change. At least there are the reindex warnings which should be easily fixable, but for the rest...

File: doc/source/changes/version_0_27.rst.inc [hunk 1]
```diff
@@ -23,7 +23,8 @@
 New features
 ------------
 
-* added a feature (see the :ref:`miscellaneous section <misc>` for details).
+* added possibility to show only rows with differences when comparing arrays or session through the function `compare`
```
Comment (gdementen) [inline doc/source/changes/version_0_27.rst.inc]: arrays or session**s** through the `compare` function

**Comment by alixdamman** [comment]
@gdementen  OK to merge?

**Comment by gdementen** [comment]
sure, I already approved it earlier

**Pull request merged**